string id
geometry_msgs/PoseStamped pose
geometry_msgs/Transform surface_transform
